src/app/accounts/account-limit.ts

```diff
@@ -0,0 +1,82 @@
+import { volumesForAccountId } from "@app/payments/helpers"
+import { getMidPriceRatio } from "@app/shared"
+import { getAccountLimits, getDealerConfig, ONE_DAY } from "@config"
+import { AccountLimitsType } from "@domain/accounts"
+import { AccountLimitsVolumes } from "@domain/accounts/limits-volume"
+import { InvalidAccountLimitTypeError } from "@domain/errors"
+import { LedgerService } from "@services/ledger"
+
+export const accountLimit = async ({
```

Comment:
```suggestion
export const remainingLimit = async ({
```

maybe this would be more appropriate?